app/javascript/src/components/editor/ScriptImporter.svelte

```diff
@@ -64,15 +70,21 @@
     <div class="modal__content">
       Copy your snippet from inside of Overwatch and paste it in here to convert your current snippet to this project.
 
-      <textarea class="form-input form-textarea form-textarea--extra-small mt-1/4" bind:value />
+      <textarea
+        class="form-input form-textarea form-textarea--extra-small mt-1/4"
+        bind:value
+        use:submittable
+        on:submit={submit}
+      />
```

Comment:
The closing tag should still be on the same line as the same attribute

```suggestion
        on:submit={submit} />
```

Reply:
I see a few cases of this. Might be worth investigating if eslint can detect that?
![image](https://github.com/Mitcheljager/workshop.codes/assets/6181929/405e33e2-d539-429f-b8a0-5cc31b6a6245)

Reply:
The ones that aren't false positives are all by you I believe. I think ESLint is able to detect this, but I'm not sure of the exact setting.